content/05.use-cases-c.md

```diff
@@ -48,10 +48,10 @@ Because the repositories are grouped, members can reference and contribute to ea
 Contributors can be assembled into teams within an organization, which allows administrators to assign roles, tasks, and repository modification permissions to organization members.
 Whereas access to repositories is usually assigned to individual contributors, Organizations facilitate the management of access permissions by allowing teams to be granted access to specific repositories.
 This ensures repositories with sensitive information remain as restricted as needed, while others stay open and accessible to selected member groups.
-The organization structure also allows for easy issue tracking and discussions related to research content and progress.
+The organization structure also allows for issue tracking and discussions related to research content and progress.
 
-As an example, GitHub Organizations are particularly well-suited to house documents and projects within a laboratory, such as research compendia, codes of conduct, protocols, training documents, and other such documents that evolve collaboratively over time and are relevant to many colleagues. 
-In this way, teams can have full ownership of repositories within an organization, while ensuring that these materials stay accessible to the laboratory after people have moved on or lost their copies of project data.
-This application extends to research centres, which may include several distinct projects that remain linked to institutions, such as the German Centre for Integrative Biodiversity Research (iDiv, <https://github.com/idiv-biodiversity>).
-The team organizing the hackathon which inspired this paper used a GitHub Organization (SORTEE-Github-Hackathon, <https://github.com/SORTEE-Github-Hackathon>) to keeping track of all materials as ideas developed and took shape in one virtual space.
-Organizations are also convenient for hosting learning materials, including lectures or workshops, such as the Québec Centre for Biodiversity Science R Workshop Series (QCBSRworkshops, <https://github.com/QCBSRworkshops>) or the University of Edinburgh's Coding Club (Coding Club, <https://github.com/ourcodingclub>), which may be updated by an ever-evolving group of contributors over time.
+As an example, GitHub Organizations are particularly well-suited to host documents and projects within a laboratory, such as research compendia, codes of conduct, protocols, training documents, and other relevant documents that evolve collaboratively over time. 
+In this way, teams have full ownership of repositories within an organization, while ensuring that these materials stay accessible to the laboratory after people have moved on or when locally-stored data is lost.
+This application extends to research centres, which may include several distinct projects that remain linked to institutions [_e.g._, the German Centre for Integrative Biodiversity Research (iDiv, <https://github.com/idiv-biodiversity>)].
```

Comment:
In markdown shouldn't this be with parentheses on the outside. Typically i would do parentheses on the outside then square brackets on the interior (i.e. before iDiv) but I'm not sure if that would mess with the markdown formatting. So maybe we just keep both as parentheses?

 (_e.g._, the German Centre for Integrative Biodiversity Research (iDiv, <https://github.com/idiv-biodiversity>)).

Reply:
OK sounds good, I'll just leave as is